src/routes/docs/advanced/platform/dev-keys/+page.markdoc

```diff
@@ -0,0 +1,87 @@
+---
+layout: article
+title: Dev keys
+description: Bypass Appwrite rate limits and CORS errors in your development environment with Appwrite Dev keys.
+---
+
+Dev keys are secrets used by Appwrite [Client SDKs](/docs/sdks#client) to avoid abuse limits in testing. They are meant to be used specifically in development environments, where they hold several developer experience-related benefits:
+
+- Appwrite rate limits and CORS errors are bypassed
+- Configurable expiration date with 1 day, 7 days, 30 day options as well as manual date selection
+
+This is highly beneficial in scenarios where you are repeatedly sending the same requests to Appwrite in a short period of time, such as manual or E2E testing and checks in your CI/CD pipeline.
```

Comment:
didn't we decide to remove this from usecases example?

Reply:
Not as far as I know

When did that change?